guides/micronaut-jaxrs-jdbc/java/src/main/java/example/micronaut/PetResource.java

```diff
@@ -0,0 +1,29 @@
+package example.micronaut;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import java.util.List;
+import java.util.Optional;
+
+@Path("/pets") // <1>
+class PetResource {
+
+    private final PetRepository petRepository;
+
+    PetResource(PetRepository petRepository) { // <2>
+        this.petRepository = petRepository;
+    }
+
+    @GET // <3>
+    List<NameDto> all() { // <4>
```

Comment:
this and `byName` work in Micronaut because the generated bean class is in the same package, but IntelliJ complains that the two methods should be public. I'd make the scope `public` so readers don't think there's a real problem